src/components/map/index.js

```diff
@@ -1,53 +1,28 @@
 import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
 import marker from './icon';
-import { useEffect, useState } from 'react';
+import Target from '../target';
 
 import 'leaflet/dist/leaflet.css';
 import './styles.scss';
 
-const MapView = () => {
-  const [position, setPosition] = useState({
-    where: [],
-  });
-  const geoOptions = {
-    enableHighAccuracy: true,
-    timeout: 5000,
-    maximumAge: 0,
-  };
-  const geolocationEnabled = () => {
-    return navigator.geolocation.getCurrentPosition(
-      position => {
-        geoSuccess(position);
-      },
-      () => {},
-      geoOptions
-    );
-  };
-
-  const geoSuccess = position => {
-    setPosition({
-      where: [position.coords.latitude, position.coords.longitude],
-    });
-  };
-
-  useEffect(() => {
-    return geolocationEnabled();
-  }, []);
-
+const MapView = ({ currentPosition, sendLatLng, targets }) => {
   return (
-    position.where?.length === 2 && (
-      <MapContainer
-        className="map__container"
-        center={position.where}
-        zoom="20"
-        scrollWheelZoom={true}
-      >
-        <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
-        <Marker position={position.where} icon={marker}>
-          <Popup>You are here.</Popup>
-        </Marker>
-      </MapContainer>
-    )
+    <>
+      {currentPosition.where?.length === 2 ? (
```

Comment:
WDYT if you do 
`currentPosition.where?.length === 2 && (`
justo to avoid returning `null`if you don't meet the condition.

Reply:
done